### DISCUSSION TITLE
Icon to represent an Agent in a contact center

### DISCUSSION THREAD

**Discussion by elainelylee** [twilio-labs/paste#970] [Request: Icon] [answered]
EIP and potentially Flex needs an icon to represent an Agent and one to represent the Customer in our tables and potentially avatars that don't have an image nor full name data. 

The available avatar for "user" would work for the Customer icon.

I've found this headphone icon in Streamline that may work for the Agent. It looks different enough at a small size from the user profile icon.

![streamline-icon-meeting-headphones@140x140](https://user-images.githubusercontent.com/6718764/100177622-45b6c280-2e87-11eb-9623-a569a5d1fd6b.png)

We're looking at mid December time frame.

**Comment by elainelylee** [comment]
I found a headset streamline icon in the Twilio Presentation deck template. Does this mean brand already has this icon and can we add it to Paste?

![image](https://user-images.githubusercontent.com/6718764/100283245-fec9db00-2f21-11eb-80af-a7b1d4467843.png)

**Comment by richbachman** [comment]
Hi @elainelylee 

I'm unsure about the brand version, but we'll get a ticket added to our backlog for this.

If you're comfortable with it, we would love for you to contribute the icon using the guidelines here: https://paste.twilio.design/icons/how-to-add-an-icon/ 

Let us know if you have any questions. Thanks!

**Comment by nicedisguise** [comment]
I agree with the suggested icons to differentiate between customer and agent. I'd simplify the streamline version a bit as some of the detail is a bit unnecessary and will look potentially busy in small sizes (namely, I'd remove the two 'bumps' at the side of the headphones)

**Comment by corinne-nagel** [comment, marked as answer]
hey @elainelylee , the agent icon was added here: https://paste.twilio.design/icons
